.github/dependabot.yml

```
@@ -9,19 +9,23 @@ updates:
       - 'pr: dependencies'
     versioning-strategy: increase
     groups:
-      csstools-parser:
+      csstools:
         patterns: ['@csstools/*']
-        exclude-patterns: ['@csstools/selector-specificity'] # unrelated to other parser packages
       changesets:
```

[suggest] Let's keep this group because `@csstools/*` group update has a problem on Dependabot side. See https://github.com/dependabot/dependabot-core/issues/7646

```suggestion
        exclude-patterns: ['@csstools/selector-specificity'] # unrelated to other parser packages
      changesets:
```

@romainmenke Feel free to comment if you have any thoughts on this.

---

I agree to keep it as is.

I hope that they eventually fix that bug so that we can remove this exception :)